problem5/calculate.c

```diff
@@ -36,6 +36,12 @@ int total(int numbers[], int length) {
 
 
 bool areStringsEqual(const char* first, const char* second) {
+
+    int strcmp(const char* first, const char* second);
+    int result;
```

Comment:
So here, we want to declare the `result` before we actually use it. We can use `result` and assign it the result of the `strcmp` function. That would look something like this:

```c
int result;
result = strcmp(first, second);
```